includes/class-gs-faq-shortcode.php

```diff
@@ -48,6 +49,7 @@ function shortcode( $atts ) {
 
 		$args = array(
 			'orderby'        => 'post__in',
+			'order'			 => $a['order'],
```

Comment:
```suggestion
			'order'		=> $a['order'],
```
The extra space here should be removed.